lib/pull.ml

```diff
@@ -50,6 +50,47 @@ let pre_pull_clean_up ~full ~duniverse_dir duniverse =
         Bos.OS.Dir.delete ~must_exist:false ~recurse:true
           Fpath.(duniverse_dir / dir))
 
+let duniverse_documentation =
+  {|\
```

Comment:
Great idea adding this README :))

I think this little `\` might be a typo? I was using the upstream version of `opam-monorepo` (https://github.com/patricoferris/dream-multicore-repro/blob/main/duniverse/README.md) and noticed it.

Reply:
Indeed, this syntax does not work with `{|` quoted strings! Good catch, let's open an issue!